app/src/main/kotlin/com/jakewharton/sa4p/ui/MainUi.kt

```diff
@@ -33,7 +37,7 @@ fun MainUi(model: MainModel) {
 			},
 		) {
 			Box(modifier = Modifier.padding(it)) {
-				PendingUrls(model.pendingUrls)
+				PendingUrls(model.pendingUrls.toPersistentList())
```

Comment:
This conversion to an immutable list should be done when creating the model at the presenter layer, not as part of UI rendering.